tot/data_processing/scaler.py

```diff
@@ -0,0 +1,66 @@
+from dataclasses import dataclass
+
+import numpy as np
+import pandas as pd
+
+
+@dataclass
+class Scaler:
+    transformer: object
+
+    def __post_init__(self):
+        is_transformer_valid = (
```

Comment:
Very nice! Can we link the url of the sklearn tranformers here to point out a selection of transformers that work?